src/_cffi_src/openssl/ssl.py

```diff
@@ -470,17 +470,19 @@
     return 1;
 }
 #endif
+
 /* Added in 1.0.2 but we need it in all versions now due to the great
    opaquing. */
-#if OPENSSL_VERSION_NUMBER < 0x10002001L || defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
I don't remember specifically, but I know we had to change a definition much like this one to avoid a bug last year. You can probably dig it up if you hunt through the history, but that's a lot of work.

I'm extremely reluctant to merge a change like this though because I'm almost certain it will cause breakage as is.